export-to-csv.ts

```diff
@@ -153,12 +153,19 @@ export class ExportToCsv {
      * Create Body
      */
     private _getBody() {
-        const keys = Object.keys(this._data[0]);
         for (var i = 0; i < this._data.length; i++) {
+            const keys = Object.keys(this._data[i]);
             let row = "";
+
             for (let keyPos = 0; keyPos < keys.length; keyPos++) {
                 const key = keys[keyPos];
-                row += this._formatData(this._data[i][key]) + this._options.fieldSeparator;
+                const cellData = this._data[i][key];
+
+                if (cellData !== "" && !cellData) {
```

Comment:
Prevent to render undefined when cell is empty and allow to render empty string ""

Reply:
won't we miss the field separator if you do `continue`?

why not go with this approach in `_formatData`?
```ts
if (!data ) {
  return "";
}
```

like in this PR:
https://github.com/alexcaza/export-to-csv/pull/26